### PR TITLE
Merged-PR proofreading

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**

### CODE REVIEW

**Review by xu-yuan1 — CHANGES_REQUESTED** [review]
For installing-openvino-docker-windows.md and installing-openvino-pip.md and other installation topics, please do NOT change the structure as it's agreed by developers. You can directly copy the file from releases/22/1.

File: docs/install_guides/installing-openvino-pip.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ You can install both OpenVINO™ Runtime and OpenVINO Development Tools through
...
+
+|Operating System (64-bit)                         | [Python Version](https://www.python.org/downloads/) (64-bit)                   |
+|--------------------------------------------------|--------------------------------------------------------------------------------|
+|macOS 10.15.x versions                            | 3.6, 3.7, 3.8                                                                  |
+|Red Hat Enterprise Linux 8                        | 3.6, 3.8                                                                       |
+|Ubuntu 18.04 long-term support (LTS)              | 3.6, 3.7, 3.8                                                                  |
+|Ubuntu 20.04 long-term support (LTS)              | 3.6, 3.7, 3.8                                                                  |
+|Windows 10                                        | 3.6, 3.7, 3.8                                                                  |
+
 For system requirements and troubleshooting, see <https://pypi.org/project/openvino/>.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:21]: You can revise this sentence if you want to add System Requirements above.

File: docs/install_guides/installing-openvino-pip.md [hunk 2]
```diff
@@ -73,13 +85,110 @@ python -c "from openvino.runtime import Core"
 
 If installation was successful, you will not see any error messages (no console output).
 
+
+@sphinxdirective
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:89]: The following steps are repeating the 5 steps above.

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 3]
```diff
@@ -27,148 +29,227 @@ This guide provides steps for creating a Docker image with Intel® Distribution
...
+   - The container host must have GPU running display drivers of version WDDM 2.5 or higher.
+- GPU requirement for OpenVINO: 
+   - Intel Graphics Driver for Windows of version 15.65 or higher.
 - [Docker isolation mode requirements](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/hyperv-container):
-  - Windows host and container version tags must match.
-  - [Windows host and container isolation process support](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility).
+   - Windows host and container version tags must match.
+   - [Windows host and container isolation process support](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility).
 
 ## Installation Flow
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-docker-windows.md:42]: Changes starting from here need be reverted. We agreed on having a simple overview of the flow first and then the detailed steps.

**Review by xu-yuan1 — APPROVED** [review]

**Comment by sgolebiewski-intel** [comment]
Moved to #11750

**Pull request closed without merging**